package.json

```diff
@@ -157,7 +157,7 @@
     "pm2-runtime": "./bin/pm2-runtime"
   },
   "dependencies": {
-    "async": "1.5",
+    "async": "^2.4.1",
```

Comment:
Yeah I am not sure why some of the tests are failing, looking at the async interface nothing looks non-kosher, so need to figure that out.

Should I update the `CHANGELOG.md` with the change to async?